types/types.go

```diff
@@ -150,6 +150,10 @@ func BytesToAddress(b []byte) Address {
 	return a
 }
 
+func AddressToBytes(a Address) []byte {
```

Comment:
Unused

Reply:
Removed in the next PR, a42d46541be2da9a1ad487aec790e151ecd06fad